example/redash_trino_cluster/main.tf

```diff
@@ -0,0 +1,277 @@
+locals {
+  nomad_datacenters = ["dc1"]
+  nomad_namespace   = "default"
+  trino_datastore = jsonencode({
+    catalog : "hive",
+    host : "127.0.0.1",
+    port : 8080,
+    schema : "default",
+    username : "trino"
+  })
+}
+
+module "redash" {
+  source = "../.."
+
+  # redash
+  service         = "redash"
+  host            = "127.0.0.1"
+  port            = 5000
+  container_image = "gitlab-container-registry.minerva.loc/datainn/redash-rabbit-edition:latest"
```

Comment:
Hm. Jeg har jo laget dette imaget, så skal absolutt ikke klage, men å bruke et image i som ikke er public i et open source prosjekt er vel litt fy fy. Burde vi publisere redash-rabbit-edition på dockerhub elns?

Reply:
Jeg er litt uenig. `example/redash_trino_cluster` er kun et eksempel på hvordan man kan bruke modulen, og oppgir dermed eksempel på hvilke verdier variable kan ha. I dette eksempelet må vi bruke en "egen-kombinert" image fordi Redash foreløpig ikke støtter Trino. Dette bør komme frem i dokumentasjonen til eksempelet (som er på todo-listen min). 

Jeg er forøvrig ikke uenig i at vi kan vurdere å publisere image opp til Docker Hub.